src/Resources/templates/htmlTableOutput.phtml

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>SensioLabs DeprecationDetector</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
OK, here it is the needed CSS and HTML: https://gist.github.com/javiereguiluz/8ef2512c805e1b73db30  Feel free to ask me anything you may need. Thanks!